src/main/java/seedu/address/logic/parser/ChangeThemeCommandParser.java

```diff
@@ -16,10 +19,22 @@
      */
     public ChangeThemeCommand parse(String userInput) throws ParseException {
         String trimmedArgs = userInput.trim();
+        String filepath = "C:\\repos\\addressbook-level4\\src\\main\\resources\\view\\" +  trimmedArgs + ".css";
+        File check = new File("C:/repos/addressbook-level4/src/main/resources/view/" + trimmedArgs + ".css");
```

Comment:
can just put filepath + trimmedArgs + .css

Reply:
do not use hard coded paths, it will not work on other machines.

Reply:
oh yea, just put relative path